src/components/LoadingPage.js

```diff
@@ -0,0 +1,13 @@
+import "./LoadingPage.css";
+import { Link } from "react-router-dom";
+import { ReactComponent as LoadingPageName } from "../icons/eten.svg";
+
+export default function LoadingPage() {
+  return (
+    <Link to={"/signInPage"} className="loading-grid">
```

Comment:
I see in general you are using `camelCase` for paths. You should know that by default urls are case insensitive. Hence i'd recommend to use `kebab-case` instead. 

https://reactrouter.com/web/api/Route/sensitive-bool